test/no-special-in-object-member.test.js

```diff
@@ -27,6 +27,11 @@ ruleTester.run('no-special-in-object-members', noSpecialInObjectMember, {
 		},
 		{
 			code: `let object = {'simpleMethod':() => 'method'}`
+		},{
+			code: ''
+		},
+		{
+			code:'let i = 0'
 		}
```

Comment:
Let's add also an empty object `{}`

Reply:
```Js
{}
```
This is not an empty object expression, but ES6 block. We need an object particularly